### PR TITLE
Initial turn in for quality lab, added google blog steps, tests appear to be be passing 

### PR DESCRIPTION
**Pull request by melanie-oneill**

### CODE REVIEW

File: e2e/src/test/resources/edu/cscc/topics/quality/e2e/google-cloud.feature [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Feature: Google Cloud Blog test
+
+Given I am on the Google Cloud homepage
```
Comment (jschmersal-cscc) [inline e2e/src/test/resources/edu/cscc/topics/quality/e2e/google-cloud.feature]: This doesn't run when I do a `mvn install`.  Hint: It's missing a line - look at the existing simple.feature file.

File: e2e/src/test/java/edu/cscc/topics/quality/e2e/GoogleCloudSteps.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        WebDriverManager.firefoxdriver().setup();
+        driver = new FirefoxDriver();
+    }
+
+    @Given("^I am on the Google Cloud homepage$")
+    public void givenIAmOnTheGoogleCloudHomepage() {
+       driver.navigate().to("https://cloud.google.com");
+    }
+
+    @When("^I click th eblog link$")
```
Comment (jschmersal-cscc) [inline e2e/src/test/java/edu/cscc/topics/quality/e2e/GoogleCloudSteps.java]: Even when I make it so I can run the feature file, this `When` isn't invoked.

File: unit/src/test/java/edu/cscc/topics/quality/unit/SampleJUnitTest.java [hunk 3]
```diff
@@ -16,20 +16,13 @@ public void simpleTest() {
...
+    @Test
     public void thisShouldRunAsATest() {
         assertTrue(new SimpleJoiner().join("|", "a", "b").length() == 3);
     }
 
-    /** Please modify the code and add a test to support passing <code>null</code> for parts
-     *
-     *  In this scenario the code should just return the <code>base</code> with no separators
-     */
-    // TODO: add this test case
```
Comment (jschmersal-cscc) [inline unit/src/test/java/edu/cscc/topics/quality/unit/SampleJUnitTest.java]: You're missing this TODO - make a test the if you call `join(".", "a", null)` meets the expectations.

**Comment by jschmersal-cscc** [comment]
This is still missing the unit test I mentioned you were supposed to add in the prior pull request.  Also, you google scenario is still missing the `Scenario` line, so it still doesn't run.
